java/core/src/java/org/apache/orc/impl/RunLengthIntegerWriterV2.java

```diff
@@ -286,7 +285,9 @@ private void writePatchedBaseValues() throws IOException {
     // find the number of bytes required for base and shift it by 5 bits
     // to accommodate patch width. The additional bit is used to store the sign
     // of the base value.
-    final int baseWidth = utils.findClosestNumBits(min) + 1;
+    int baseBitsNum = utils.findClosestNumBits(min);
```

Comment:
The root cause here is because we add 1 here, so `baseWidth` ranges from 2 to 65,  and `baseBytes` ranges from 0 to 8 which does not fit into 3 bits.